apps/files/src/components/AcceptInviteBar.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div id="accept-invite-bar" class="oc-app-bar">
+    <oc-grid flex gutter="small">
+      <div class="uk-flex-1">
+        <div class="uk-flex">
+          <oc-breadcrumb id="files-breadcrumb" :items="breadcrumbs" v-if="showBreadcrumb" home></oc-breadcrumb>
+        </div>
+        <span class="uk-flex uk-flex-middle" v-if="!showBreadcrumb">
+          <oc-icon v-if="pageIcon" :name="pageIcon" class="uk-margin-small-right" />
+          <h1 class="oc-page-title" v-text="pageTitle" />
+        </span>
+        <span v-else-if="showBreadcrumb">
+          <h1 class="oc-visually-hidden" v-text="pageTitle" />
+        </span>
+      </div>
+    </oc-grid>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapState } from 'vuex'
+import pathUtil from 'path'
+
+export default {
+  computed: {
+    ...mapGetters(['getToken', 'configuration']),
+    ...mapGetters('Files', ['publicLinkPassword']),
+    ...mapState(['route']),
+    showBreadcrumb () {
+      return (
+        this.$route.name === 'public-files' || this.$route.name === 'files-list'
+      )
+    },
+    pageIcon () {
+      return this.$route.meta.pageIcon
+    },
+    pageTitle () {
+      const title = this.route.meta.pageTitle
+      return this.$gettext(title)
+    },
+
+    breadcrumbs () {
```

Comment:
Are breadcrumbs here needed? It doesn't look to me like there would be a need for multiple routes.

Reply:
I don't think they are. I remove them.